src/main/app/claims/models/uploadedDocument.ts

```diff
@@ -24,4 +24,8 @@ export class UploadedDocument implements Serializable<UploadedDocument> {
     }
     return this
   }
+
+  getId (): string {
+    return this.documentManagementURI.slice(11)
```

Comment:
this is wrong way of doing, if they change the url, then it will fail. best is use their config url and and get their length and then slice